src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -6362,7 +6362,7 @@ void AutoCalcDOASControlStrategy(EnergyPlusData &state)
 
     int ZoneSizIndex;
     bool ErrorsFound;
-
+    bool headerAlreadyPrinted = false;
     ErrorsFound = false;
     for (ZoneSizIndex = 1; ZoneSizIndex <= state.dataSize->NumZoneSizingInput; ++ZoneSizIndex) {
         if (state.dataSize->ZoneSizingInput(ZoneSizIndex).AccountForDOAS) {
```

Comment:
`auto &zoneSizingInput = state.dataSize->ZoneSizingInput(ZoneSizIndex);` (also love shortening `Size` to `Siz`)